autoload/clap/provider/common_history.vim

```diff
@@ -0,0 +1,33 @@
+" Author: liuchengxu <xuliuchengxlc@gmail.com>
+" Description: Common utilities for command/search history.
```

Comment:
Do not place the utils code in the provider directory, which will be considered as a provider incorrectly by `:Clap providers`. You can put these code in `autoload/clap/util.vim`.

Reply:
Ah ... Okay!

Reply:
Done!